jest.config.ts

```diff
@@ -5,7 +5,6 @@ const jestConfig = {
     '^src/(.*)': ['<rootDir>/src/$1'],
     '^test/(.*)': ['<rootDir>/test/$1'],
     '@eppo(.*)': '<rootDir>/node_modules/@eppo/$1',
-    '^uuid$': '<rootDir>/node_modules/uuid/dist/index.js',
```

Comment:
@felipecsl FYI - removing uuid now that we are on v11

Reply:
nice, thanks!